src/compass/utils/h5_helpers.py

```diff
@@ -614,16 +614,19 @@ def corrections_to_h5group(parent_group, burst, cfg, rg_lut, az_lut,
         Meta('los_solid_earth_tides', ds.GetRasterBand(4).ReadAsArray(),
              f'Solid Earth tides (range) {desc}',
              {'units': 'meters'}),
+        Meta('los_ionospheric_delay', ds.GetRasterBand(5).ReadAsArray(),
```

Comment:
NOTE: The most recent branch as of now does not seem to have ionosphere LUT written into the product, which seem to happen on #112. Here I add the ionosphere back. Also note that the band ID of the LUTs are adjusted accordingly.